saltproc/depcode.py

```diff
@@ -16,14 +17,24 @@ class Depcode(ABC):
     population, active, and inactive cycles. Contains methods to read template
     and output files, and write new input files for the depletion code.
 
+    Attributes
+    -----------
+    param : dict of str to type
```

Comment:
It's not clear what this str or type is.

Reply:
I've expanded the docstring.